RNDeviceInfo.podspec

```diff
@@ -15,7 +17,28 @@ Pod::Spec.new do |s|
   s.tvos.deployment_target = '10.0'
 
   s.source       = { :git => "https://github.com/react-native-device-info/react-native-device-info.git", :tag => "v#{s.version}" }
-  s.source_files  = "ios/**/*.{h,m}"
+  s.source_files  = "ios/**/*.{h,m,mm}"
+
+  if fabric_enabled
+    folly_compiler_flags = '-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1 -Wno-comma -Wno-shorten-64-to-32'
```

Comment:
there's an easier way btw: https://github.com/react-native-datetimepicker/datetimepicker/blob/4bd45866508738dd7511264f8ea12cbe64e07dda/RNDateTimePicker.podspec#L19

Reply:
We'd have to do it in all the libs we migrated then.

Reply:
@WoLewicki, @j-piasecki  actually, you don't need to wrap `install_modules_dependencies` into `if fabric_enabled` - it can be used without the condition: https://reactnative.dev/docs/next/the-new-architecture/backward-compatibility-turbomodules#ios

I missed that in the example I gave

Reply:
@vonovak That's true, but IIRC it's been added in RN 0.71 so that would mean dropping support for versions below that.